test/image/mocks/gl2d_parcoords_out-of-range_selected-all.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Renders currently as:

![image](https://user-images.githubusercontent.com/6675409/59610259-052a1580-90e7-11e9-975d-79f7910ce67b.png)

Reply:
@etpinard Thanks for documenting this one.